doc/edge-metrics-v2.md

```diff
@@ -0,0 +1,43 @@
+| `edgehub_reported_properties_update_duration_seconds_count` | `iothub` (IoT Hub Name)<br> `edge_device` (IoT Edge device name)<br> `target` (Operation target)<br> `id` (Module ID)                                                                | Measurement count                                                                                                                                                                                        | summary |
+| `edgehub_reported_properties_update_duration_seconds`       | `iothub` (IoT Hub Name)<br> `edge_device` (IoT Edge device name)<br> `target` (Operation target)<br> `id` (Module ID)<br> `quantile`(Percentile [50, 90, 95, 99, 99.9, 99.99])       | P50, P90, P95, P99, P99.9 and P99.99 time taken to update reported properties. Values may be reported as `NaN`  if no new measurements are reported for a certain  period of time (currently 10 minutes) | summary |
+|                                                             |                                                                                                                                                                                      |                                                                                                                                                                                                          |         |
+|                                                             |                                                                                                                                                                                      |                                                                                                                                                                                                          |         |
+|                                                             |                                                                                                                                                                                      |                                                                                                                                                                                                          |         |
+|                                                             |                                                                                                                                                                                      |                                                                                                                                                                                                          |         |
+
+### Collecting
+
+Metrics will be available for collection at `http://edgeHub:9600/metrics` on the IoT Edge module network or `http://localhost:9600/metrics` if port mapped to the host at the default port number.
```

Comment:
Did you try mapping the port to the host? We haven't exposed the 9600 port in the exposed ports. So just mapping the port to host may not work.
https://github.com/Azure/iotedge/blob/f93c6f3f217a3ee2a6bc0ba291c936db7aef7f50/edge-hub/docker/linux/amd64/Dockerfile#L27

Reply:
It works after you add `ExposedPorts` to createOptions. I can add that step.